### DISCUSSION TITLE
Header Text & Some Other Doubt

### DISCUSSION THREAD

**Discussion by srjtmitra5** [forntoh/LcdMenu#74] [Q&A]
Hello,
1. 
MenuItem mainMenu[] = {ItemHeader(),
                       MenuItem("Start service"),
                       MenuItem("Connect to WiFi"),
                       MenuItem("Settings"),
                       MenuItem("Blink SOS"),
                       MenuItem("Blink random"),
                       ItemFooter()};

In this the first Line of ItemHeader(), can we make the provision of passing a String or Char Arrary to it where it will display the header on top of the Menu and if null the menu will start from 1st row like the way it is happening now?

2. Can we attach an event to a particular MenuItem which will open a customized LCD Screen once I click the menu item and on pressing ESC/ENT it will take be back to the Menu?

Any help will be highly appreciated.

Regards,
Srijit

**Comment by forntoh** [comment]
1. Header and footer are not actual Menu items, their purpose is to tell the navigation system if it has reached the top or the bottom of the menu respectively. For what you wan, you need to implement a custom functionality yourself.
2. Have a look at [this discussion topic](https://github.com/forntoh/LcdMenu/discussions/45) it can be of help.
